src/views/transactions/components/transaction-form/transaction-form.view.jsx

```diff
@@ -154,6 +154,11 @@ function TransactionForm({
   function isContinueDisabled() {
     if (!isAmountInputDirty) {
       return true;
+    } else if (
+      account.token.USD === null &&
+      (transactionType === TxType.Transfer || transactionType === TxType.Exit)
```

Comment:
It should show the correct error

Reply:
Do you mean to show an alert with an explication? The user doesn't reach this page in a normal flow

Reply:
They could be in Account Details and click Transfer. This would skip the Account Selector, right?

Reply:
You don't see the button in account details neither in the selector if the usd comes as null